doc/installing/development-py35.md

```diff
@@ -0,0 +1,191 @@
+
+```bash
+## RedHat
+sudo yum install git scl-utils scl-utils-build
+
+## CentOS/SL
+sudo yum install git centos-release-scl
+
+sudo yum install rh-python35
+scl enable rh-python35 bash
+```
+
+### macOS
+*This procedure refers to the official Python distribution from [python.org](https://python.org). If you are using a different python (from brew, macports, conda) you may need to adapt the following commands.*
+
+#### Xcode
+
+You must install [Xcode](https://itunes.apple.com/app/xcode/id497799835?mt=12) first.
+
+#### Python 
+
+You need to download Python from [python.org](https://python.org): https://www.python.org/ftp/python/3.5.3/python-3.5.3-macosx10.6.pkg
+
+## Build the development environment
+
+Let's create a working dir called openquake first
+
+```bash
+mkdir $HOME/openquake && cd $HOME/openquake
+```
+
+then build a development environment using python *virtualenv*
+
+```bash
+python3 -m venv oqenv 
+source oqenv/bin/activate
+```
+
+## Install the code
+
+Inside the *virtualenv* (the prompt shows something like `(oqenv)user@myhost:~$`) upgrade `pip` and `setuptools` first
+
+```bash
+pip install -U pip setuptools
+```
+
+### Download the OpenQuake source code
+
+```bash
+mkdir src && cd src
+git clone https://github.com/gem/oq-engine.git
+```
+
+### Install OpenQuake 
+
+It's strongly recommended to install Python dependencies using our Python wheels distribution: all the external dependencies (`geos`, `proj4`, `hdf5`, `blas`, and many other) are already included as pre-compiled binaries and are tested before every release.
+
+```bash
+# For Linux
+pip install -r oq-engine/requirements-py35-linux64.txt
+```
+
+```bash
+# For macOS
+pip install -r oq-engine/requirements-py35-macos.txt
+```
+
+```bash
+pip install -e oq-engine/
+```
+To install extra features see [1](#note1). If your system does not support the provided binary dependencies you'll need to manually install them, using tools provided by your python distribution [2](#note2).
+
+Now it is possible to run the OpenQuake Engine with `oq engine`. Any change made to the `oq-engine` code will be reflected in the environment.
+
+Continue on [How to run the OpenQuake Engine](../running/unix.md)
+
+### Sync the source code with remote
+
+You can pull all the latest changes to the source code running
+
+```bash
+cd oq-engine
+git pull
+cd ..
+```
+
+## Loading and unloading the development environment
+
+To exit from the OpenQuake development environment type `deactivate`. Before using again the OpenQuake software the environment must be loaded back running `source oqenv/bin/activate`(assuming that it has been installed under 'oqenv'). For more information about *virtualenv* and its you see http://docs.python-guide.org/en/latest/dev/virtualenvs/
```

Comment:
I am not sure if ```loaded back``` sounds good to a native English speaker. Otherwise you might say ```reloaded```
Please check the last sentence. Probably it can be reduced to ```For more information about *virtualenv*, see http://docs.python-guide.org/en/latest/dev/virtualenvs/```